dev-packages/e2e-tests/verdaccio-config/config.yaml

```diff
@@ -122,12 +122,6 @@ packages:
     unpublish: $all
     # proxy: npmjs # Don't proxy for E2E tests!
 
-  '@sentry/replay':
-    access: $all
-    publish: $all
-    unpublish: $all
-    # proxy: npmjs # Don't proxy for E2E tests!
```

Comment:
I was able to just delete this (instead of renaming), as there is `'@sentry-internal/*'` further down.